api/services/elastigroup/aws/schemas/elastigroup-scale-down.yaml

```diff
@@ -0,0 +1,120 @@
+      The metric statistics to return.
+    enum: [ average , sum , sampleCount , maximum , minimum , percentile ]
+    example: "average"
+  extendedStatistic:
+    type: string
+    description: |
+      Percentile statistic. Valid values: p0.1 - p100
+    example: "p1.5"
+  unit:
+    type: string
+    description: |
+      The unit for the alarm's associated metric.
+    enum: [ seconds , microseconds , milliseconds , bytes , kilobytes , megabytes , gigabytes , terabytes , bits , kilobits , megabits , gigabits , terabits , percent , count , bytes/second , kilobytes/second , megabytes/second , gigabytes/second , terabytes/second , bits/second , kilobits/second , megabits/second , gigabits/second , terabits/second , count/second , none ]
+    example: "percent"
+  threshold:
+    type: integer
+    description: |
+      The value against which the specified statistic is compared
+    example: 90
+  action:
+    type: object
+    example: "adjustment"
+    description: |
+      The action to take when scale up is needed.
+      A mandatory field, that should be filled with one of the following:
+      "adjustment", "percentageAdjustment", "setMaxTarget", "setMinTarget", "updateCapacity".
+    properties:
+      type:
+        type: string
+        description: |
+          The type of the action to take when scale up is needed.
+        enum: [ adjustment , updateCapacity setMinTarget , percentageAdjustment ]
+        example: "updateCapacity"
+      adjustment:
+        type: integer
+        description: |
+          The number / percentage associated with the specified adjustment type.
+          Required if using `adjustment` or `percentageAdjustment` as action type
+        example: 1
+      minTargetCapacity:
+        type: integer
+        description: |
+          The number with the target capacity needed.
+          Required if using `setMinTarget` as action type
+        example: 1
+      target:
+        type: integer
+        description: |
+          The desired number of instances. Required if using `updateCapacity` as action type and neither `minimum` nor `maximum` are not defined.
+        example: 1
+      minimum:
+        type: integer
+        description: |
+          The lower limit number of instances that you can scale down to. Optional, required if using `updateCapacity` as action type and neither `target` nor `maximum` are not defined
+        example: 0
+      maximum:
+        type: integer
+        description: |
+          The upper limit number of instances that you can scale up to. Optional, required if using `updateCapacity` as action type and neither `target` nor `minimum` are not defined
+        example: 5
+  namespace:
+    type: string
+    description: |
+      The namespace for the alarm's associated metric.
+    default: "AWS/EC2"
+  dimensions:
+    type: array
+    description: |
+      The dimensions for the alarm's associated metric.
+      If the user mentioned name as instanceId, there is NO value
```

Comment:
This sentence is not clear. Where there is no value? also, no need to use the word 'user'